docs/migration-v8.md

```diff
@@ -492,6 +492,47 @@ ResiliencePipeline<HttpResponseMessage> pipelineT = new ResiliencePipelineBuilde
 ```
 <!-- endSnippet -->
 
+## Migrating timeout policy
+
+> [!NOTE]
+> In v8, the timeout resilience strategy does not support pessimistic timeouts because they can cause thread-pool starvation and non-cancellable background tasks. To address this, you can use [this workaround](https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#cancelling-uncancellable-operations) to make the action cancellable. If there's significant demand for pessimistic timeout, we might consider reintroducing it in the future.
```

Comment:
If there's enough demand, people will ask without being prompted to. I think saying this might prompt to people to ask even if it's a mild inconvenience.

```suggestion
> In v8, the timeout resilience strategy does not support pessimistic timeouts because they can cause thread-pool starvation and non-cancellable background tasks. To address this, you can use [this workaround](https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#cancelling-uncancellable-operations) to make the action cancellable.
```

Reply:
True.